### DISCUSSION TITLE
Setting up Jetbot with Docker - Help!

### DISCUSSION THREAD

**Discussion by bobbobbobby** [NVIDIA-AI-IOT/jetbot#458] [Q&A]
Hi,

I'm completely new to using Jetson Nano and the Jetbot, so I'm here to learn! I followed the instructions on this site (https://jetbot.org/v0.4.3/software_setup/docker.html) to try to set up my Jetbot, but I get the following errors when I run ./enable.sh. I'm on the latest release of Jetpack and my L4T version is 32.5.1. I've updated my config.sh file based on what I found on previous discussions.

> Synchronizing state of docker.service with SysV service script with /lib/systemd/systemd-sysv-install.
> Executing: /lib/systemd/systemd-sysv-install enable docker
> Unable to find image 'jetbot/jetbot:display-0.4.3-32.5.1' locally
> docker: Error response from daemon: manifest for jetbot/jetbot:display-0.4.3-32.5.1 not found: manifest unknown: manifest unknown.
> See 'docker run --help'.
> Unable to find image 'jetbot/jetbot:jupyter-0.4.3-32.5.1' locally
> docker: Error response from daemon: manifest for jetbot/jetbot:jupyter-0.4.3-32.5.1 not found: manifest unknown: manifest unknown.
> See 'docker run --help'.

Any troubleshooting help would be appreciated. Thanks!

**Comment by santaimpersonator** [comment]
You'll want to modify the `configure.sh` script as mentioned here: https://github.com/NVIDIA-AI-IOT/jetbot/issues/406#issuecomment-837006950

**Reply by Usimian** [reply]
I've been having the same problems.  It seems if you start with a new Jetson nano and follow the setup instructions to the letter...it doesn't work.

I even watched the video (with Jim) and when I get to the part where he uses "./run_from_ngc.sh", there's no such file.

If this was someone's pet project, then fine but it's linked right on the NVIDIA website as a working example.

I think they're doing a disservice to the community to have detailed instructions that simply don't work.  Frustration is abundant.

**Reply by santaimpersonator** [reply]
@Usimian I am not familiar with the video you mentioned (link?), but I would assume that you could comment/provide feedback to the who ever produced it for any issues.

* There's already a pull request (https://github.com/NVIDIA-AI-IOT/jetbot/pull/456) to fix the modification I mentioned. I assume it will be part of their next release. 
* If you are using an image that implements the docker container for the jetbot code, I would recommend the instructions on [jetbot.org ](http://jetbot.org/master/software_setup/docker.html) to get started.
* Otherwise, I don't understand why you replied to my comment.

**Reply by Usimian** [reply]
Sorry, I was trying to respond to the OP.  I've gotten a lot of it to work by trial and error (plus 30 years of software development!).  I'm more concerned that a newbie will try these projects and give up.

Thanks for responding.
